pwndbg/commands/one_gadget.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import argparse
+import shutil
+
+import pwndbg.color.message as M
+import pwndbg.commands
+import pwndbg.gdblib.one_gadget
+import pwndbg.glibc
+from pwndbg.commands import CommandCategory
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.RawTextHelpFormatter,
+    description="""Show one_gadget
```

Comment:
The command has been renamed to `onegadget`

Reply:
@lebr0nli perhaps also rename the file to `onegadget.py`